nextflow_schema.json

```diff
@@ -626,6 +685,17 @@
                     "description": "Specify a maximum number of reads a hit must have to be retained in the ganon report.",
                     "help_text": "Specify the maximum number of reads or percentage of counts a hit must have against a taxon to be retained. To specify a maximum percentage, specify between 0 and 1 (e.g. 0.9 for 90%), and more than 1 to specify a hard count cut off (e.g. 10000 for maximum of 10,000 reads).\n\n> Modifies tool parameter(s):\n- ganon report: `--max-count`",
                     "fa_icon": "fas fa-filter"
+                },
+                "run_sylph": {
```

Comment:
Is that meant to be in this PR?

Reply:
No! Not sure how this happened :/ Thank you for spotting this, I am updating the PR.